projects/valtimo/object-management/src/lib/services/object-management.service.ts

```diff
@@ -80,6 +80,16 @@ export class ObjectManagementService {
     );
   }
 
+  putSearchListColumns(
+    ownerId: string,
+    request: Array<SearchListColumn>
+  ): Observable<Array<SearchListColumn>> {
+    return this.http.put<Array<SearchListColumn>>(
+      `${this.valtimoEndpointUri}v1/search/list-column/${ownerId}/search-list-columns`,
```

Comment:
This endpoint doesn't accept an array

Reply:
https://github.com/valtimo-platform/valtimo-backend-libraries/pull/657/files

it does now